primitives/src/condition.rs

```diff
@@ -163,6 +163,13 @@ impl Condition {
                 claims.len()
             }
         };
-        (claims_count, self.issuers.len())
+        let tfs = self
```

Comment:
This change the meaning of `complexity` fn. 
Have you double-checked that the use of this is aligned with this change?

Reply:
Double checked and made some slight adjustments.